README.md

```diff
@@ -206,6 +206,33 @@ With a file named `vault-token.ctmpl`:
 }
 ```
 
+### Default Vault Agent configuration
+
+You can set the default `agentConfig` for all units by using the `detsys.defaultAgentConfig` interface.
+
+> **NOTE**: Manually-specified unit `agentConfig`s will override _**all**_ of the the settings specified in the `detsys.defaultAgentConfig` option.
+
+```
+{
+  detsys.defaultAgentConfig = {
```

Comment:
Doable, but the asymmetry between `detsys.vaultAgent.defaultConfig` and `detsys.systemd.services.[name].vaultAgent` feels awkward to me (`vaultAgent` in two distinctly separated locations) Should we move the per-service config to `detsys.vaultAgent.systemd.services.[name]`, then, and have that as well as `detsys.vaultAgent.defaultConfig`...?

Reply:
Sounds good!

Reply:
I decided not to shorten to `defaultConfig` so that it's unambiguous what the config is applying to -- it _only_ affects the `agentConfig` of units where `agentConfig` is already unset (and not maybe also affecting the templated secrets).